### PR TITLE
Breakdown, stars and photo working 

### PR DESCRIPTION
**Pull request by positivefx**

### CODE REVIEW

File: client/src/components/RatingReviews/Breakdown/BreakdownCharacteristic.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (positivefx) [reply client/src/components/RatingReviews/Breakdown/BreakdownCharacteristic.jsx]: gotcha removed

File: client/src/components/RatingReviews/Reviews/ReviewEntryStar.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import { FaStar, FaStarHalf } from 'react-icons/fa';
+
+function ReviewEntryStar({ rating }) {
+  const [inputStar, setInputStar] = useState(rating);
+
+  // console.log(starFillPercentage(rating));
+
+  return (
+    <div>
+      {([...Array(5)]).map((star, count) => (
```
Comment (addisonhernandez) [inline client/src/components/RatingReviews/Reviews/ReviewEntryStar.jsx:12]: Can we replace this with `_.range(5)`?

File: client/src/components/RatingReviews/WriteReview/WriteReview.jsx [hunk 3]
```diff
@@ -42,6 +42,7 @@ function WriteReview({
       .then((response) => {
         console.log('review post', response);
       })
+      .then(() => setShowWriteReview(false))
```
Comment (addisonhernandez) [inline client/src/components/RatingReviews/WriteReview/WriteReview.jsx:45]: :+1:

File: client/src/components/RatingReviews/Reviews/ReviewEntryStar.jsx [hunk 4]
```diff
@@ -10,7 +10,7 @@ function ReviewEntryStar({ rating }) {
 
   return (
     <div>
-      {_.range(5).map((star, count) => (
+      {_.range(1, 6).map((count) => (
```
Comment (addisonhernandez) [inline client/src/components/RatingReviews/Reviews/ReviewEntryStar.jsx:13]: :fire:

**Review by addisonhernandez — APPROVED** [review]

**Comment by positivefx** [comment]
[Thurs, Apr 7- Daily Merge Party 🎉](https://trello.com/c/Ns3fSkSP/38-thurs-apr-7-daily-merge-party-%F0%9F%8E%89)

**Pull request merged**